src/server/controllers/model.controller.ts

```diff
@@ -1473,3 +1483,22 @@ export const updateGallerySettingsHandler = async ({
     throw throwDbError(error);
   }
 };
+
+export async function addCheckpointCoverageHandler({
+  input,
+}: {
+  input: ToggleCheckpointCoverageInput;
+}) {
+  try {
+    const affectedRows = await toggleCheckpointCoverage(input);
+    if (input.versionId) await deleteResourceDataCache(input.versionId);
```

Comment:
We'll also need to clear the cache of all version associated with a model if a model is passed in